FORMAT.md

```diff
@@ -283,13 +300,60 @@ This is represented by a path between the same [long, lat] point on different le
 ```
 ---
 
-#### Disclaimer
+## Indoor Assets
+
+Indoor Assets are optional pieces of furniture you can add to an Indoor Map.  These are a separate GeoJSON format from the main Indoor Map format that defines all the Indoor Assets for a given floor, which can then be imported using the Import Assets function of the WRLD Indoor Map Designer.
+
+Each assets GeoJSON file is expected to contain a FeatureCollection in the WGS84 (EPSG: 4326) Coordinate Reference System, and include the appropriate crs attribute. Assets are represented as GeoJSON features with Polygon or Point geometry, with the center of the Polygon or the Point position itself being used as the central point of the Asset.
+
+### Asset Properties
+
+| Attribute| Type  | Description                                                                          |
+| -------- | ----- | ------------------------------------------------------------------------------------ |
+|`id`      |integer|An identifier for the feature. Each asset must have a unique id.
+|`name`    |string |The asset's name, for example, `Desk 42`. Optional, can be null.
+|`type`    |string |Must be set to "InstancedEntity" to be recognized by the Asset Import process.
+|`instance`|string |A Prop ID matching one from our [Prop List](prop-list.html).
```

Comment:
Prop List link should go to the prop-manifest file on the CDN, I think (until we have the prop-models repo accessible).

Reply:
https://cdn-resources.wrld3d.com/props/latest/Assets/manifest.json